packages/quantic/force-app/main/default/lwc/quanticSummary/quanticSummary.js

```diff
@@ -98,23 +98,14 @@ export default class QuanticSummary extends LightningElement {
         I18nUtils.getTextBold(this.query));
   }
 
-  get duration() {
-    if (this.state?.hasDuration) {
-      const duration = this.state.durationInSeconds;
-      const labelName = I18nUtils.getLabelNameWithCount('inSeconds', duration);
```

Comment:
Was it the only place referencing this label? If yes, I guess we can remove the label as well without breaking anything.